src/pages/index.tsx

```diff
@@ -91,28 +90,24 @@ const Index: FC = () => {
             )}
         </div>
     );
+
+    const renderContent = () => {
+        if (isError) return <p>{TASKS_FETCH_ERROR_MESSAGE}</p>;
+
+        if (isLoading) return <p>Loading...</p>;
+
+        return (
+            <div className={classNames.tasksContainer}>
+                {renderTabSection()}
+                {renderTaskList()}
```

Comment:
Any reason to use function call instead of react component?

Reply:
no thanks for pointing out, let me correct it

Reply:
done